### ISSUE TITLE
Freshly compiled Godot scales funny on OS X 10.10

### ISSUE THREAD

**Issue by philiplb** [okamstudio/godot#1118]
On a retina MacBook Pro with I just compiled Godot, commit 8c4dd8de3919b789eac1a0f030563008bdcd44f5 and started directly the resulting godot.osx.tools.32.
The scaling is a bit funny. It scales the same way in the started game.
Environment:
- OS X 10.10
- XCode 6.1.1
- Scons 2.3.4
- g++:
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/usr/include/c++/4.2.1
Apple LLVM version 6.0 (clang-600.0.56) (based on LLVM 3.5svn)
Target: x86_64-apple-darwin14.0.0
Thread model: posix

See attached screenshot.
![bildschirmfoto 2015-01-05 um 01 48 58](https://cloud.githubusercontent.com/assets/1672238/5608253/d77c8628-947d-11e4-9f33-a5d74223632e.png)

**Comment by reduz** [comment]
I have no idea how to disable retina on freshly compiled binaries. If you
put them inside the project (the app folder as specificied in the build
instructions) it works, but I'm sure that this could be somehow fixed in
os_osx.mm. I think this used to work properly before

**Comment by philiplb** [comment]
Ok, thanks for the answer. As soon as I started the .app version, it just worked fine. Interesting.

(Some lines might be added in the wiki about how to copy the binary:
mkdir tools/Godot.app/Contents/MacOS
cp bin/godot.osx.tools.32 tools/Godot.app/Contents/MacOS/Godot
I had to have a look at the original .app in order to figure that out. Might be a lack of knowledge how an OS X app looks and totally obvious though.)
